openslides/motions/static/js/motions/site.js

```diff
@@ -1231,6 +1245,41 @@ angular.module('OpenSlidesApp.motions.site', [
     }
 ])
 
+.controller('ChangeRecommendationUpdateCtrl', [
+    '$scope',
+    'MotionChangeRecommendation',
+    'ChangeRecommendationForm',
+    'change',
+    function ($scope, MotionChangeRecommendation, ChangeRecommendationForm, change) {
+        $scope.alert = {};
+        $scope.model = change;
```

Comment:
You have to use `$scope.model = angular.copy(change);` - like in [MotionUpdateCtrl](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/static/js/motions/site.js#L1364). Otherwise you can change and cancel a change recommendation and the changes are displayed... Then you need a reload to be in sync with server data. So please update this line.